Sources/CoreData/Combine/CombineSingleObjectRepository.swift

```diff
@@ -0,0 +1,34 @@
+//
+// AppwiseCore
+// Copyright © 2022 Appwise
+//
+
+import CoreData
+import Combine
+
+@available(iOS 13.0, *)
+public protocol CombineSingleObjectRepository {
```

Comment:
What does this type add? Can't we just add the `refresh()` function as an extension to the existing protocol? And have a default implementation, that wraps the call to `refresh(then:)`